syntax = "proto3";

package bayesmix;

enum AlgorithmId {
    UNKNOWN_ALGORITHM = 0;
    Neal2 = 1;
    Neal3 = 2;
    Neal8 = 3;
    BlockedGibbs = 4;
    SplitMerge = 5;
    Slice = 6;
}
